### PR TITLE
Resource View Toolbar

### PR DESCRIPTION
**Pull request by EvenCode**

### CODE REVIEW

**Comment by Compro-Prasad** [comment]
It seems like you have commited extra files.

File: src/resources-view_impl.cxx [hunk 1]
```diff
@@ -22,8 +22,100 @@ ResourcesViewImpl::ResourcesViewImpl (
   {
     parent->remove (*m_top_widget);
   }
+
```
Comment (Compro-Prasad) [inline src/resources-view_impl.cxx]: Keep this file.

File: src/resources-view_impl.h [hunk 2]
```diff
@@ -34,6 +34,17 @@ class ResourcesViewImpl : public ResourcesView
     m_handler = handler;
   }
```
Comment (Compro-Prasad) [inline src/resources-view_impl.h]: Keep this file.

File: src/resources/resources-view.glade [hunk 3]
```diff
@@ -2,26 +2,187 @@
 <!-- Generated with glade 3.20.0 -->
 <interface>
   <requires lib="gtk+" version="3.20"/>
```
Comment (Compro-Prasad) [inline src/resources/resources-view.glade]: Keep this file.

File: src/resources/wbs-view.glade [hunk 4]
```diff
@@ -31,6 +31,7 @@
               <object class="GtkToolButton" id="tb-cut">
                 <property name="visible">True</property>
```
Comment (Compro-Prasad) [inline src/resources/wbs-view.glade]: Keep this file.

**Comment by Compro-Prasad** [comment]
Apart from "Keep this file." remove every file. You can use `git rm <file1> <file2>` and commit again.

**Comment by EvenCode** [comment]
The CMakeFiles folder you mean ?

**Pull request closed without merging**